### PR TITLE
Match -M to actual resource requests.

### PR DESCRIPTION
**Pull request by tmooney**
Some bad requests were copy+pasted around over time... I found a copy of this `-M` limit in a version of the Samtools command from 2009.

### CODE REVIEW

File: lib/perl/Genome/Model/Build/ReferenceSequence/AnnotationIndex.pm [hunk 1]
```diff
@@ -81,7 +81,7 @@ sub __display_name__ {
 
 sub required_rusage {
     # override in subclasses
-    # e.x.: "-R 'span[hosts=1] rusage[tmp=50000:mem=12000]' -M 1610612736";
+    # e.x.: "-R 'span[hosts=1] rusage[tmp=50000:mem=12000]' -M 12000000";
     ''
 }
```
Comment (ebelter) [inline lib/perl/Genome/Model/Build/ReferenceSequence/AnnotationIndex.pm]: While we are at it, should the 'select' statement be added to the examples?

**Comment by ebelter** [comment]
+1

**Comment by nnutter** [comment]
:+1:

File: lib/perl/Genome/Model/Build/ReferenceSequence/AnnotationIndex.pm (hunk 1 above)
Comment (jasonwalker80) [inline lib/perl/Genome/Model/Build/ReferenceSequence/AnnotationIndex.pm]: Doesn't Flow remove all select statements?

**Pull request merged**